### PR TITLE
clientv3: fix README, add error handling example

### PR DESCRIPTION
**Pull request by gyuho**
/cc @xiang90 @heyitsanthony

### CODE REVIEW

File: clientv3/example_kv_test.go [hunk 1]
```diff
@@ -40,6 +41,34 @@ func ExampleKV_put() {
...
+			log.Fatalf("ctx is canceled by another routine: %v", err)
+		case context.DeadlineExceeded:
+			log.Fatalf("ctx is attached with a deadline and it exceeded: %v", err)
+		case rpctypes.ErrEmptyKey:
+			log.Fatalf("client-side error: %v", err)
+		default:
+			log.Fatalf("bad cluster endpoints, which are not etcd servers: %v", err)
+		}
+	}
+	// client-side error: etcdserver: key is not provided
```
Comment (heyitsanthony) [inline clientv3/example_kv_test.go]: `// Output: client-side error: etcdserver: key is not provided` so it runs as a test?

File: clientv3/example_kv_test.go [hunk 2]
```diff
@@ -40,6 +41,34 @@ func ExampleKV_put() {
...
+
+	ctx, cancel := context.WithTimeout(context.Background(), requestTimeout)
+	_, err = cli.Put(ctx, "", "sample_value")
+	cancel()
+	if err != nil {
+		switch err {
+		case context.Canceled:
+			log.Fatalf("ctx is canceled by another routine: %v", err)
+		case context.DeadlineExceeded:
+			log.Fatalf("ctx is attached with a deadline and it exceeded: %v", err)
```
Comment (heyitsanthony) [inline clientv3/example_kv_test.go]: `deadline that is exceeded`?

**Comment by heyitsanthony** [comment]
lgtm aside from minor nits

**Comment by gyuho** [comment]
ok fixed. merging after test passes. Thanks!

**Pull request merged**